src/chapters/08/main.tex

```diff
@@ -489,10 +556,11 @@ \section{Solving with R}\label{sec:solving-with-R}
 }
 \end{Rin-no-test}
 
-Finally the following function uses them all to assemble a coefficients matrix.
+Finally the following function uses all previous functions
+to assemble a coefficients matrix.
 It loops though the parameters for each constraint row required, uses the
 appropriate function to create the row of the coefficients matrix, sets the
-appropriate inequality direction ($\leq$, $=$, $\geq$), and the value of the
+appropriate inequality direction (\(\leq$, $=$, $\geq\)), and the value of the
```

Comment:
does this compile as expected?

Reply:
Yeah but not correctly, I automated a subsitution and broke it. I'll fix.

Reply:
I'll fix when I go through and test the code :)